Foundation/NSTask.swift

```diff
@@ -7,6 +7,7 @@
 // See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 
+#if !os(Android) // not available
```

Comment:
Had a quick look at removing the #if !os(Android), got the following:

Foundation/NSTask.swift:95:2: warning: unknown platform 'Android' for attribute 'available'
@available(Android, unavailable)
 ^